editor/editor_file_system.h

```diff
@@ -159,11 +159,22 @@ class EditorFileSystem : public Node {
 		EditorFileSystemDirectory::FileInfo *new_file = nullptr;
 	};
 
+	class ScannedDirectory {
+		friend class EditorFileSystem;
```

Comment:
You can instead change this class to `struct` and put it under `private:`, though it's a cosmetic change.

Reply:
I'll change for a struct to be like the other structs in EditorFileSystem but what do you mean by "put it under private:", there is no 'private:' section in EditorFileSystem, I can add one still.

Reply:
I changed it for a struct.